stable/external-dns/templates/clusterrole.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/{{ .Values.rbac.apiVersion }}
 kind: ClusterRole
 metadata:
   labels: {{ include "external-dns.labels" . | indent 4 }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
this should be indented, but the include labels should be on a newline

Reply:
My fail) Sorry.